pkgs/applications/misc/pdfstudio/default.nix

```diff
@@ -1,82 +1,78 @@
-{ stdenv,
-  lib,
-  makeWrapper,
-  fetchurl,
-  dpkg,
-  makeDesktopItem,
-  copyDesktopItems,
-  autoPatchelfHook,
-  gst_all_1,
-  sane-backends,
-  xorg,
-  gnome2,
-  alsa-lib,
-  libgccjit,
-  jdk11
-  }:
+{ stdenv
+, lib
+, fetchurl
+, libgccjit
+, dpkg
+, makeDesktopItem
+, copyDesktopItems
+, autoPatchelfHook
+, sane-backends
+, jdk11
+}:
 
-let
-  year = "2021";
-  major = "1";
-  minor = "2";
-in stdenv.mkDerivation rec {
+# See also package 'pdfstudioviewer'
+# Differences are ${pname}, Download directory name (PDFStudio / PDFStudioViewer),
+# sha256, and libgccjit (not needed for PDFStudioViewer)
+let year = "2021";
+in
+stdenv.mkDerivation rec {
   pname = "pdfstudio";
-  version = "${year}.${major}.${minor}";
-  autoPatchelfIgnoreMissingDeps = true;
+  version = "${year}.1.2";
+  autoPatchelfIgnoreMissingDeps = false;
```

Comment:
Setting this to a non-null value actually enables it: https://github.com/NixOS/nixpkgs/blob/2ed91b734904f854a33ca939b2ee7310c8980c4d/pkgs/build-support/setup-hooks/auto-patchelf.sh#L357

It set to false by default, so the line can just be removed.

Reply:
I removed `autoPatchelfIgnoreMissingDeps = false;`